package.json

```diff
@@ -3,30 +3,35 @@
   "version": "0.1.9",
   "description": "Utility functions for working with Quadbins",
   "license": "MIT",
+  "type": "module",
```

Comment:
Note — It might be safest to publish this on a v0.2.0 release rather than v0.1.10. I think adding `"type": "module"` is one of those things that's 99% safe but should not be done in a patch release to be conservative if others depend on the package.